chirp/inference/classify/classify.py

```diff
@@ -190,35 +192,78 @@ def write_inference_csv(
     threshold: dict[str, float] | None = None,
     exclude_classes: Sequence[str] = ('unknown',),
     include_classes: Sequence[str] = (),
+    row_size: int = 1_000_000,
+    format: str = 'parquet',
 ):
-  """Write a CSV file of inference results."""
+  """Write inference results."""
+  
+  if format != 'parquet' and format != 'csv':
+    raise ValueError('Format must be either "parquet" or "csv"')
+  
+  if format == 'parquet':
+    if output_filepath.endswith('.csv'):
+      output_filepath = output_filepath[:-4]
+    if not output_filepath.endswith('.parquet'):
```

Comment:
This second-guessing of the user-intention from the extension and format args is a bit cumbersome.

Maybe we should get the extension from the output file and use that instead of an arg? (and complain if it's not one of our accepted types.)

Then we would have:
```
if output_filepath.endswith('.parquet'):
  format = 'parquet'
elif output_filepath.endswith('.csv'):
  format = 'csv'
else:
  raise ValueError(...)
```
which saves an argument and ~12 lines of code.